src/app/models/navigation.ts

```diff
@@ -0,0 +1,14 @@
+export const MenuList = [
+  {
+    title: 'Squads',
+    href:'squads'
+  },
+  {
+    title: 'Rankings',
+    href:'rankings'
+  },
+  {
+    title: 'Games',
+    href:'games'
+  }
+]
```

Comment:
We can also make an interface that would look like this that we can export to go with it:
```suggestion
]

export interface MenuListItem {
     title: string,
     href: string
     }
```

And then when you use it you say that you have an array of menu list items like this:

`Array<MenuListItem>`

Reply:
what does that do?